controller/controller.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"iotdashboard/dbmanager"
+	"iotdashboard/utils"
+)
+
+type ControllerService struct {
+	PSQL      dbmanager.DBManager
+	TokenUtil utils.TokenUtil
+}
+
+func NewController() (ControllerService, error) {
```

Comment:
Why does this return a value and not a pointer?

Reply:
No good reason. Changed all `New____` functions to return pointers